src/components/Header.test.js

```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import '@testing-library/jest-dom/extend-expect';
+import Header from './Header';
```

Comment:
## OPTIONAL 
* I mark this as optional because it wasn't mentioned by previous CRs but I strongly suggest you fix the errors I get when I ran tests below 👇 is a screenshot. For HINTS you can install `redux-mock-store` and wrap your component in a provider because your component `Header` is trying to access the store here is an example code 
![image](https://user-images.githubusercontent.com/52497246/227702410-fc497d00-a306-4e7d-9ec4-d2e5b17bc130.png)


![image](https://user-images.githubusercontent.com/52497246/227701878-b1c96cd5-1d75-443b-85e8-dd2457e17e1c.png)